packages/stores/src/ui-config/manage-liquidity/bond-liquidity.ts

```diff
@@ -129,6 +136,16 @@ export class ObservableBondLiquidityConfig extends UserConfig {
           const curDuration = dayjs.duration({
             milliseconds: durationMs,
           });
+          const lockedUserShares = queryLockedCoin.getLockedCoinWithDuration(
+            this.poolDetails.poolShareCurrency,
+            curDuration
+          ).amount;
+
+          const totalShares = this.poolDetails.pool.totalShare;
+          const poolTvl = this.poolDetails.totalValueLocked;
+          const userShareValue = poolTvl.mul(
```

Comment:
It would be more concise to call this `userLockedShareValue`

Reply:
Also, it doesn't appear to be used anywhere?

Reply:
This var is used on the bond card to display the user's locked share value